build.py

```diff
@@ -38,6 +38,7 @@
 use_plugin("source_distribution")
 
 use_plugin("python.unittest")
+use_plugin("python.pytest")
```

Comment:
Because I now only two way to force pybuilder to install any package (in this case - pytest, because pytest_plugin tests use it):
1) use build_depends_on('pytest')
2) use plugin_depends_on('pytest') and call use_plugin into build.py (otherwise plugin will not be used and pytest will not be installed)
Any other solution?